### PR TITLE
lang: Russian translation 

### PR DESCRIPTION
**Pull request by KaneDbD**
Translation improved, added more lines

### CODE REVIEW

File: lang/ru.xml [hunk 1]
```diff
@@ -27,31 +27,52 @@
...
+в зависимости от его размера и производительности вашей системы.</app_delete_note>
+		<addcont_delete>Удалить DLC?</addcont_delete>
+		<license_delete>Удалить лицензию?</license_delete>
+		<save_delete>Удалить эти сохранённые данные?</save_delete>
+		<eligible>Доступно</eligible>
+		<ineligible>Не доступно</ineligible>	
 		<level>Уровень</level>
-		<name>Название</name>
-		<trophy_earning>Полученные призы</trophy_earning>
+		<name>Назв.</name>
```
Comment (Zangetsu38) [inline lang/ru.xml:69]: revert

**Comment by Zangetsu38** [comment]
@Croden1999 if you can review it, and compare with official string please

File: lang/ru.xml [hunk 2]
```diff
@@ -27,31 +27,52 @@
...
+		<app_delete_note>Удаление приложения может занять некоторое время
+в зависимости от его размера и производительности вашей системы.</app_delete_note>
+		<addcont_delete>Удалить DLC?</addcont_delete>
+		<license_delete>Удалить лицензию?</license_delete>
+		<save_delete>Удалить эти сохранённые данные?</save_delete>
+		<eligible>Доступно</eligible>
+		<ineligible>Не доступно</ineligible>	
 		<level>Уровень</level>
-		<name>Название</name>
-		<trophy_earning>Полученные призы</trophy_earning>
```
Comment (Zangetsu38) [inline lang/ru.xml:49]: revert, is correct

File: lang/ru.xml [hunk 3]
```diff
@@ -27,31 +27,52 @@
...
+		<name>Назв.</name>		
+		<trophy_earning>Получение призов</trophy_earning>	
 		<parental_controls>Родительский контроль</parental_controls>
-		<updated>Обновлен</updated>
+		<updated>Обновлено</updated>
```
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ru.xml:72]: Обновлен is correct, revert

File: lang/ru.xml [hunk 4]
```diff
@@ -27,31 +27,52 @@
...
-		<trophy_earning>Полученные призы</trophy_earning>
+		<name>Назв.</name>		
+		<trophy_earning>Получение призов</trophy_earning>	
 		<parental_controls>Родительский контроль</parental_controls>
-		<updated>Обновлен</updated>
+		<updated>Обновлено</updated>		
 		<size>Размер</size>
 		<version>Версия</version>
-		<last_time_used>Время последнего использования</last_time_used>
+		<last_time_used>Время послед. использования</last_time_used>
```
Comment (Zangetsu38) [inline lang/ru.xml:75]: revert

File: lang/ru.xml [hunk 5]
```diff
@@ -264,41 +328,149 @@ depending on its size and your hardware.</app_delete_note>
...
+	</home_screen>
+
 	<indicator>
 		<app_added_home>Приложение добавлено на начальный экран.</app_added_home>
 		<delete_all>Удалить все</delete_all>
 		<notif_deleted>Уведомления будут удалены.</notif_deleted>
 		<install_failed>Не удалось выполнить установку.</install_failed>
 		<install_complete>Установка завершена.</install_complete>
-		<installing>Идет установка...</installing>
+		<installing>Идёт установка...</installing>
```
Comment (Zangetsu38) [inline lang/ru.xml:357]: revert

File: lang/ru.xml [hunk 6]
```diff
@@ -230,31 +294,31 @@ depending on its size and your hardware.</app_delete_note>
...
+				<new_saved_data>Новые сохранённые данные</new_saved_data>
 				<saving_complete>Сохранение завершено.</saving_complete>
 				<save_the_data>Сохранить данные?</save_the_data>
-				<saving>Идет сохранение...</saving>
-				<warning_saving>Идет сохранение...
+				<saving>Идёт сохранение...</saving>
+				<warning_saving>Идёт сохранение...
 Не выключайте систему и не закрывайте приложение.</warning_saving>
-				<overwrite_saved_data>Перезаписать сохраненные данные?</overwrite_saved_data>
+				<overwrite_saved_data>Перезаписать сохранённые данные?</overwrite_saved_data>
```
Comment (Zangetsu38) [inline lang/ru.xml:321]: revert all

File: lang/ru.xml [hunk 7]
```diff
@@ -230,31 +294,31 @@ depending on its size and your hardware.</app_delete_note>
 			<delete>
 				<cancel_deleting>Отменить удаление?</cancel_deleting>
 				<deletion_complete>Удаление завершено.</deletion_complete>
-				<delete_saved_data>Удалить эти сохраненные данные?</delete_saved_data>
+				<delete_saved_data>Удалить эти сохранённые данные?</delete_saved_data>
```
Comment (Zangetsu38) [inline lang/ru.xml:297]: revert

File: lang/ru.xml [hunk 8]
```diff
@@ -186,31 +206,36 @@ depending on its size and your hardware.</app_delete_note>
...
-		<saved_data name="Сохраненные данные">
-			<delete>Выбранные сохраненные данные будут удалены.</delete>
-			<no_saved_data>Отсутствуют сохраненные данные.</no_saved_data>
+		<saved_data name="Сохранённые данные">
+			<delete>Выбранные сохранённые данные будут удалены.</delete>
+			<no_saved_data>Отсутствуют сохранённые данные.</no_saved_data>
 		</saved_data>
 		<theme>Тема</theme>
-		<free_space>Cвободное</free_space>
+		<free_space>Cвободно</free_space>
```
Comment (Zangetsu38) [inline lang/ru.xml:238]: revert all

File: lang/ru.xml [hunk 9]
```diff
@@ -61,14 +82,13 @@ depending on its size and your hardware.</app_delete_note>
 Код ошибки: {}</an_error_occurred>
 		<cancel>Отмена</cancel>
 		<delete>Удалить</delete>
-		<file_corrupted>Файл поврежден.</file_corrupted>
+		<file_corrupted>Файл повреждён.</file_corrupted>
```
Comment (Zangetsu38) [inline lang/ru.xml:85]: revert

File: lang/ru.xml [hunk 10]
```diff
@@ -27,31 +27,52 @@
...
+		<shaders_log>Лог шейдеров</shaders_log>
 		<update_history>Хронология обновлений</update_history>
 		<history_version>Версия {}</history_version>
 		<information>Информация</information>
 		<app_delete>Это приложение и все связанные с ним данные, в том числе сохраненные данные, будут удалены.</app_delete>
-		<app_delete_note>Deleting a application may take a while
-depending on its size and your hardware.</app_delete_note>
-		<save_delete>Удалить эти сохраненные данные?</save_delete>
-		<eligible>Допущен</eligible>
-		<ineligible>Не допущен</ineligible>
```
Comment (Zangetsu38) [inline lang/ru.xml:46]: revert

File: lang/ru.xml [hunk 11]
```diff
@@ -264,41 +328,149 @@ depending on its size and your hardware.</app_delete_note>
...
-		<installing>Идет установка...</installing>
+		<installing>Идёт установка...</installing>
 		<no_notif>Уведомления отсутствуют.</no_notif>
 		<trophy_earned>Вы получили приз!</trophy_earned>
 	</indicator>
 
 	<initial_setup>
 		<back>Назад</back>
-		<completed_setup>Теперь начальная установка завершена.
+		<completed_setup>Начальная настройка завершена.
```
Comment (Zangetsu38) [inline lang/ru.xml:364]: revert

File: lang/ru.xml [hunk 12]
```diff
@@ -264,41 +328,149 @@ depending on its size and your hardware.</app_delete_note>
...
-		<download_font_package>Download Font Package</download_font_package>
-		<install_firmware_file>Install Firmware File</install_firmware_file>
-		<completed>Completed.</completed>
+		<install_highly_recommended>Настоятельно рекомендуется 
+установить прошивку и шрифты.</install_highly_recommended>
+		<installed>Установлено:</installed>
+		<download_firmware>Скачать прошивку</download_firmware>
+		<download_font_package>Скачать пакет шрифтов</download_font_package>
+		<install_firmware_file>Установить прошивку</install_firmware_file>
+		<completed>Готово.</completed>
```
Comment (Zangetsu38) [inline lang/ru.xml:374]: revert

File: lang/ru.xml [hunk 13]
```diff
@@ -308,14 +480,14 @@ depending on its size and your hardware.</app_delete_note>
 				</information>
 				<delete>Эта тема будет удалена.</delete>
 			</theme>
-			<start_screen name="Экран запуска">
+			<start_screen name="Начальный экран">
```
Comment (Zangetsu38) [inline lang/ru.xml:483]: revert

File: lang/ru.xml [hunk 14]
```diff
@@ -325,62 +497,71 @@ depending on its size and your hardware.</app_delete_note>
 				<mm_dd_yyyy>ММ/ДД/ГГГГ</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Формат времени">
-				<clock_12_hour>12-часовой формат</clock_12_hour>
-				<clock_24_hour>24-часовой формат</clock_24_hour>
+				<clock_12_hour>12-часовой форм.</clock_12_hour>
+				<clock_24_hour>24-часовой форм.</clock_24_hour>
```
Comment (Zangetsu38) [inline lang/ru.xml:501]: revert both

File: lang/ru.xml [hunk 15]
```diff
@@ -325,62 +497,71 @@ depending on its size and your hardware.</app_delete_note>
...
 	<trophy_collection>
+		<search>Поиск</search>
+		<delete_trophy>Удалить призы</delete_trophy>
+		<trophy_deleted>Информация о призах, сохранённая для этого пользователя, 
+будет удалена.</trophy_deleted>
+		<locked>Не
+разбл.</locked>
 		<details>Подробности</details>
-		<earned>Получены</earned>
+		<earned>Получено</earned>
```
Comment (Zangetsu38) [inline lang/ru.xml:539]: revert

File: lang/ru.xml [hunk 16]
```diff
@@ -325,62 +497,71 @@ depending on its size and your hardware.</app_delete_note>
...
 		<name>Название</name>
 		<no_trophies>Призы отсутствуют.
 Можно получать призы, используя приложение, поддерживающее функцию призов.</no_trophies>
-		<not_earned>Не получен</not_earned>
+		<not_earned>Не получено</not_earned>
```
Comment (Zangetsu38) [inline lang/ru.xml:543]: revert

File: lang/ru.xml [hunk 17]
```diff
@@ -325,62 +497,71 @@ depending on its size and your hardware.</app_delete_note>
...
 		<original>Исходный</original>
 		<sort>Сортировка</sort>
 		<trophies>Призы</trophies>
 		<grade>Категория</grade>
 		<progress>Выполнено</progress>
-		<updated>Обновлен</updated>
+		<updated>Новые</updated>
```
Comment (Zangetsu38) [inline lang/ru.xml:549]: revert

File: lang/ru.xml [hunk 18]
```diff
@@ -264,41 +328,149 @@ depending on its size and your hardware.</app_delete_note>
...
+			<browse_page>Просмотр 
+страницы</browse_page>
+			<browse_page_control>D-pad, л. стик влево/вправо, колёсико вверх/вниз 
+либо нажав &lt;/&gt;</browse_page_control>
+			<hide_show>Скрыть/показать кнопки</hide_show>
+			<hide_show_control>Правая кнопка мышки</hide_show_control>
+			<zoom_available>Зум при возможности</zoom_available>
+			<zoom_available_control>Два раза ЛКМ</zoom_available_control>
+			<scroll_zoom>Прокрутка в режиме зума</scroll_zoom>
+			<scroll_zoom_control>Колёсико вверх/вниз</scroll_zoom_control>
```
Comment (Zangetsu38) [inline lang/ru.xml:456]: delete this, need change this, zoom no exist anymore

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

File: lang/ru.xml (hunk 12 above)
Comment (CoffeeBrewer64) [reply lang/ru.xml:374]: lgtm, no need to revert

File: lang/ru.xml (hunk 10 above)
Comment (bookmist) [reply lang/ru.xml:46]: Are you sure here? Previous translation looks nonsense on information screen.
изображение

File: lang/ru.xml (hunk 8 above)
Comment (CoffeeBrewer64) [reply lang/ru.xml:238]: no need revert, is all right

Comment (Zangetsu38) [reply lang/ru.xml:238]: > no need revert, is all right

i uysing official psvita string, so is wrong

**Comment by Zangetsu38** [comment]
> It looks good to me. Sometimes it was needed to shorten texts to make it fit into screen or prevent overlapping.

is wrong, no set short name, that is other issue
set short name, also set short name for information screen, no sense

File: lang/ru.xml (hunk 8 above)
Comment (CoffeeBrewer64) [reply lang/ru.xml:238]: please could you show me where the string is used officially? on discord if preferred

Comment (Zangetsu38) [reply lang/ru.xml:238]: ? i just said here, is officiel string from psvita

**Comment by bookmist** [comment]
> is wrong, no set short name, that is other issue
> set short name, also set short name for information screen, no sense

изображение

Here first two letters on <last_time_used> string are out of screen. What could be done here?

**Comment by Zangetsu38** [comment]
> > is wrong, no set short name, that is other issue
> > set short name, also set short name for information screen, no sense
> 
> изображение
> 
> Here first two letters on <last_time_used> string are out of screen. What could be done here?

So is gui issue, no lang relate

File: lang/ru.xml [hunk 19]
```diff
@@ -27,28 +27,49 @@
...
-depending on its size and your hardware.</app_delete_note>
-		<save_delete>Удалить эти сохраненные данные?</save_delete>
+		<app_delete_note>Удаление приложения может занять некоторое время
+в зависимости от его размера и производительности вашей системы.</app_delete_note>
+		<addcont_delete>Удалить DLC?</addcont_delete>
+		<license_delete>Удалить лицензию?</license_delete>
+		<save_delete>Удалить эти сохранённые данные?</save_delete>
 		<eligible>Допущен</eligible>
-		<ineligible>Не допущен</ineligible>
+		<ineligible>Не допущен</ineligible>
```
Comment (Croden1999) [inline lang/ru.xml:67]: wrong line

File: lang/ru.xml [hunk 20]
```diff
@@ -27,28 +27,49 @@
...
+		<license_delete>Удалить лицензию?</license_delete>
+		<save_delete>Удалить эти сохранённые данные?</save_delete>
 		<eligible>Допущен</eligible>
-		<ineligible>Не допущен</ineligible>
+		<ineligible>Не допущен</ineligible>	
 		<level>Уровень</level>
-		<name>Название</name>
-		<trophy_earning>Полученные призы</trophy_earning>
+		<name>Название</name>		
+		<trophy_earning>Полученные призы</trophy_earning>
```
Comment (Croden1999) [inline lang/ru.xml:70]: also here

File: lang/ru.xml [hunk 21]
```diff
@@ -27,28 +27,49 @@
...
-		<ineligible>Не допущен</ineligible>
+		<ineligible>Не допущен</ineligible>	
 		<level>Уровень</level>
-		<name>Название</name>
-		<trophy_earning>Полученные призы</trophy_earning>
+		<name>Название</name>		
+		<trophy_earning>Полученные призы</trophy_earning>	
 		<parental_controls>Родительский контроль</parental_controls>
-		<updated>Обновлен</updated>
+		<updated>Обновлен</updated>
```
Comment (Croden1999) [inline lang/ru.xml:72]: here

File: lang/ru.xml [hunk 22]
```diff
@@ -186,21 +206,26 @@ depending on its size and your hardware.</app_delete_note>
...
+	<compatibility name="Совместимость">
 		<states>
-			<state id = "-1">Unknown</state>
-			<state id = "0">Бесполезно</state>
-			<state id = "1">Загружается</state>
-			<state id = "2">Интро</state>
+			<state id = "-1">Неизвестно</state>
+			<state id = "0">Не запускается</state>
+			<state id = "1">Запускается</state>
+			<state id = "2">Заставка</state>
```
Comment (Croden1999) [inline lang/ru.xml:214]: wrong line

File: lang/ru.xml [hunk 23]
```diff
@@ -186,21 +206,26 @@ depending on its size and your hardware.</app_delete_note>
...
 	</compatibility>
 
+	<compile_shaders>
+		<compiling_shaders>Компиляция шейдеров</compiling_shaders>
+		<shaders_compiled>{} шейдеров скомпилировано</shaders_compiled>
+	</compile_shaders>
+
 	<content_manager name="Управление данными">
-		<search>Search</search>
+	<search>Поиск</search>
```
Comment (Croden1999) [inline lang/ru.xml:228]: here

File: lang/ru.xml [hunk 24]
```diff
@@ -311,11 +482,11 @@ depending on its size and your hardware.</app_delete_note>
 			<start_screen name="Экран запуска">
 				<image>Изображение</image>
 				<add_image>Добавить
-Изображение</add_image>
-			</start_screen>
+изображение</add_image>
+				</start_screen>
```
Comment (Croden1999) [inline lang/ru.xml:486]: revert

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by Croden1999** [comment]
Mr. @Zangetsu38, here is,

File: lang/ru.xml (hunk 19 above)
Comment (Zangetsu38) [reply lang/ru.xml:67]: Want said, revert, tiu adding tab or space here

File: lang/ru.xml (hunk 23 above)
Comment (Zangetsu38) [inline lang/ru.xml:228]:
```suggestion
	    <search>Поиск</search>
```

File: lang/ru.xml [hunk 25]
```diff
@@ -325,62 +497,71 @@ depending on its size and your hardware.</app_delete_note>
...
+						<lang id="32">Итальянская</lang>
+						<lang id="64">Голландская</lang>
+						<lang id="128">Норвежская</lang>
+						<lang id="256">Польская</lang>
+						<lang id="131072">Португальская (Бразилия)</lang>
+						<lang id="512">Португальская (Португалия)</lang>
+						<lang id="1024">Русская</lang>
+						<lang id="2048">Финская</lang>
+						<lang id="4096">Шведская</lang>
+						<lang id="524288">Турецкая</lang>
```
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ru.xml:523]: @Croden1999 you can screen on your psvita ime keyboard list in russian too ?

[the remark this answers is not in the document]
Comment (Croden1999) [reply lang/ru.xml:523]: he should revert

File: lang/ru.xml [hunk 26]
```diff
@@ -389,25 +569,46 @@ depending on its size and your hardware.</app_delete_note>
...
+		<automatic_user_login>Автоматический вход в систему</automatic_user_login>
 	</user_management>
 
 	<vita3k_update name="Обновление Vita3K">
  		<new_version_available>Доступна новая версия Vita3K.</new_version_available>
 		<back>Назад</back>
- 		<cancel_update_resume>Отменить обновление?
+		<cancel_update_resume>Отменить обновление?
 Если отменить обновление, то при следующем его запуске Vita3K начнет загрузку с этой точки.</cancel_update_resume>
-		<downloading>Идет загрузка...
```
Comment (Zangetsu38) [inline lang/ru.xml:400]: revert this is good

File: lang/ru.xml [hunk 27]
```diff
@@ -389,25 +569,46 @@ depending on its size and your hardware.</app_delete_note>
...
-После завершения загрузки Vita3K автоматически перезапустится и установит новую обновление.</downloading>
+		<downloading>Идёт загрузка...
+После завершения загрузки Vita3K автоматически перезапустится и установит новую версию.</downloading>
```
Comment (Zangetsu38) [inline lang/ru.xml:581]: but keep here is good change

File: lang/ru.xml (hunk 25 above)
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/ru.xml:523]: so revert all :)

File: lang/ru.xml (hunk 20 above)
Comment (Zangetsu38) [reply lang/ru.xml:70]: revert change

File: lang/ru.xml (hunk 27 above)
Comment (Zangetsu38) [inline lang/ru.xml:581]: reverrt both

File: lang/ru.xml [hunk 28]
```diff
@@ -349,14 +520,20 @@ depending on its size and your hardware.</app_delete_note>
 						<lang id="1024">Русский</lang>
 						<lang id="2048">Финский</lang>
 						<lang id="4096">Шведский</lang>
-						<lang id="524288">турецкий</lang>
+						<lang id="524288">Турецкий</lang>
```
Comment (bookmist) [inline lang/ru.xml:523]: OMG. It's obvious mistake, but I found same mistake in the original vita firmware. So revert. :(

**Review by bookmist — APPROVED** [review]

File: lang/ru.xml [hunk 29]
```diff
@@ -281,24 +365,111 @@ depending on its size and your hardware.</app_delete_note>
...
+			<exit_livearea>Выйти из Live Area</exit_livearea>
+			<exit_livearea_control>Нажать Esc или кружок</exit_livearea_control>
+			<manual_help>Справка по мануалу</manual_help>
+			<browse_page>Просмотр 
+страницы</browse_page>
+			<browse_page_control>D-pad, л. стик влево/вправо, колёсико вверх/вниз 
+либо нажав &lt;/&gt;</browse_page_control>
+			<hide_show>Скрыть/показать кнопки</hide_show>
+			<hide_show_control>Правая кнопка мышки</hide_show_control>
+			<zoom_available>Зум при возможности</zoom_available>
```
Comment (Zangetsu38) [inline lang/ru.xml:453]: I have explain, delete this zoom line

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**